PSSwagger/PSSwagger.psm1

```diff
@@ -773,8 +778,7 @@ function ConvertTo-CsharpCode
         }
         $dependencies = Get-PSSwaggerExternalDependencies -Azure:$codeCreatedByAzureGenerator -Framework 'netstandard1'
         $microsoftRestClientRuntimeAzureRequiredVersion = if ($dependencies.ContainsKey('Microsoft.Rest.ClientRuntime.Azure')) { $dependencies['Microsoft.Rest.ClientRuntime.Azure'].RequiredVersion } else { '' }
-        $command = "Import-Module '$PSScriptRoot\PSSwaggerUtility';
```

Comment:
>Import-Module '$PSScriptRoot\PSSwaggerUtility' [](start = 20, length = 46)

Consider separating this fix if a 0.2.1 hotfix release sounds correct.

Reply:
What do you mean by "if a 0.2.1 hotfix release sounds correct."?

We should not use $PSScriptRoot for importing the PSSwaggerUtility module. I am not sure, how a hotfix will fix this without removing below line.
`Import-Module '$PSScriptRoot\PSSwaggerUtility'`

Reply:
If releasing a 0.2.1 hotfix to gallery sounds correct to resolve this breaking issue, consider **separating** this fix from the CSC changes. Otherwise, to fix this issue, we will have to release the CSC changes as well.

Reply:
This change doesn't qualify to be hotfix for PSSwagger module as this line just writes an Import-Module error in verbose stream and it wont block the cmdlet execution. I am planning to release the new versions of our modules after enabling latest AutoRest support.